### DISCUSSION TITLE
How to merge a pull request without reviewing the code ?

### DISCUSSION THREAD

**Discussion by GouriSankarPinaca** [community/community#138571] [Pull Requests]
### Select Topic Area

Question

### Body

How to merge a pull request without reviewing the code ?

**Comment by TGouriSankar** [comment]
1.     Create a new pull request in your repo and open it
2.     In the top right corner at Reviewers section, request someone to review your pull request
3.     Merge the pull request without finishing the review.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/138570#discussioncomment-10662384) for reference to why this discussion was closed and locked. Thanks!
